specification/healthcareapis/resource-manager/Microsoft.HealthcareApis/preview/2022-07-01-preview/healthcare-apis.json

```diff
@@ -4093,6 +4138,225 @@
         "modelAsString": true
       },
       "readOnly": true
+    },
+    "MedtechMeasurementProperty": {
+      "type": "object",
+      "description": "The values for an extracted measurement",
+      "properties": {
+        "name": {
+          "description": "The name of the measurement",
+          "type": "string"
+        },
+        "value": {
+          "description": "The value of the measurement",
+          "type": "string"
+        }
+      },
+      "readOnly": true
+    },
+    "MedtechMeasurement": {
+      "type": "object",
+      "description": "A measurement extracted during the Medtech device event normalization process.",
+      "properties": {
+        "type": {
+          "description": "The type of measurement being collected.",
+          "type": "string"
+        },
+        "correlationId": {
+          "description": "The id used to group values into a single observation.",
+          "type": "string"
+        },
+        "deviceId": {
+          "description": "The id of the device that collected the measurement.",
+          "type": "string"
+        },
+        "encounterId": {
+          "description": "The id of the encounter.",
+          "type": "string"
+        },
+        "occurrenceTimeUtc": {
+          "description": "The time that device collected the measurement.",
+          "type": "string",
+          "format": "date-time"
+        },
+        "properties": {
+          "type": "array",
+          "x-ms-identifiers": [],
```

Comment:
>"x-ms-identifiers": [], [](http://example.com/codeflow?start=10&length=23)

The x-ms-identifiers should not be empty, if possible to populate it. For example if here "name" property of MedtechMeasurementProperty uniqly identifies each Measurement property in the array, you can use "name" in x-ms-identifiers.